addons/spreadsheet/static/src/pivot/pivot_table.js

```diff
@@ -171,8 +180,91 @@ export class SpreadsheetPivotTable {
         return this._rows[index];
     }
 
-    getRowTitle() {
-        return this._rowTitle;
+    /**
+     * @returns {PivotCell[][]}
+     */
+    getPivotCells(includeTotal = true) {
```

Comment:
could we consider making this lazy? Each re-evaluation would go through this maybe without good reason?

Reply:
Nice catch
or just compute it in the constructor and store it (readonly property) ? It's ok to store a computed/derived value if the data is immutable/all read only